vortex-sampling-compressor/src/compressors/alp.rs

```diff
@@ -53,6 +60,7 @@ impl EncodingCompressor for ALPCompressor {
             .map(|p| {
                 ctx.auxiliary("patches")
                     .excluding(self)
+                    .including(&ALPRDCompressor)
```

Comment:
It's not excluded, rather it isn't *included* in the DEFAULT_COMPRESSOR set anymore. This is to say, we only try ALPRD on the ALP patches, rather than as a top-level encoding

Reply:
I think you still want top level alprd, you still can have top level real doubles. Albeit in tpch there’s probably only one column like that

Reply:
I think we need a way to express top-level preferences. I'd only want to take ALPRD as a top-level compression scheme if ALP failed (e.g., basically all patches)

Reply:
Right so you can try alprd in alp compressor and then potentially return alprd compressor from alp compressor. I think since we return comperssorref independently of the compressed array you can see play games with implementation

Reply:
no advantage to doing that, I think. Just reverted for now